context7.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://context7.com/schema/context7.json",
+
+    "projectTitle": "NeuroConv",
+    "description": "Convert 40+ neurophysiology data formats to NWB with automatic metadata extraction.",
+
+    "folders": [
+      "docs/"
+    ],
+
+    "excludeFolders": [
+      "src/",
+      "tests/",
+      "dockerfiles/",
+      ".github/",
+      ".venv/",
+      "build/",
+      "dist/",
+      "docs/_build/",
+      "docs/_static/"
+    ],
+
+    "excludeFiles": [
+      "CHANGELOG.md",
+      "license.txt",
+      "MANIFEST.in",
+      "pyproject.toml",
+      "CITATION.cff",
+      "codecov.yml"
```

Comment:
the Context7 project website says it does `version-specific documentation and code examples`. If that's the case, wouldn't it make sense to keep `CHANGELOG.md`?

Also, it feels like `pyproject.toml` is relevant, since agents might need to debug failures caused by missing dependencies and similar issues.